pkg/parser.go

```diff
@@ -0,0 +1,22 @@
+package parser
+
+import (
+	"encoding/json"
+
+	"github.com/asyncapi/parser/pkg/hlsp"
+)
+
+var (
+	DefaultDefinitionFile = "asyncapi/2.0.0/example.yaml"
```

Comment:
IMHO, this should be on main.go. The parser should not know anything about the CLI stuff.

Reply:
:+1: